jupyterlab/labbox_ephys_widgets_jp/src/pluginWidgets.tsx

```diff
@@ -249,8 +250,24 @@ const PluginComponentWrapper: FunctionComponent<PluginComponentWrapperProps> = (
       selection: model.get('selection')
     })
   }, null)
+
+  // externalUnitMetrics
```

Comment:
We handle the externalUnitMetrics attribute in a very similar way to curation and selection attributes.